packages/shared/jest.config.js

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
Can we remove all the commented code?

The final result should be:
```js
module.exports = {
  coverageProvider: 'v8',
};
```

Reply:
if you don't want to add the extra properties, at least add the one about `setupFiles` because is causing errors when running `yarn test`